packages/core-bridge/src/helpers/abort_controller.rs

```diff
@@ -81,16 +101,6 @@ impl AbortControllerInner {
         &self,
         cx: &mut C,
     ) -> BridgeResult<Handle<'cx, JsValue>> {
-        // It is however possible that the rust-side controller might get aborted from a non-Node
-        // thread while the JS-side controller is being created on the Node thread, in which case
-        // we don't want the Rust-side thread to get blocked for the JS-side to complete
-        // instantiation. Hence, we
-        //
-        // The fact that we require a Context here means that we are running on the Node's thread,
-        // which guarantees that there can't be multiple threads calling into this function
-        // concurrently; it threfore makes no difference from the JS thread either we acquire the
-        // lock here or not.
-
         if let Some(js_counterpart) = self.js_counterpart.get() {
             // Already initialized, return the signal
             return Ok(js_counterpart.signal.to_inner(cx).upcast());
```

Comment:
Git won't let me highlight unchanged stuff, but, probably everything in here should happen under a `get_or_init` call for atomicity